dotcom-rendering/src/web/lib/liveblogAdSlots.ts

```diff
@@ -28,49 +38,57 @@ type BlockElementHeight = {
 
 /**
  * Approximations of the size each block element type will take up on the page in pixels.
- * Predictions are made for desktop, as we prefer to use a value closer to the lower bound,
- * as we would rather display too few ads than too many. Practically, this will mean that we
- * will show the the right frequency of ads on desktop and too few on smaller devices.
+ * Predictions are made for mobile viewports, as data suggests that the majority of liveblog page
```

Comment:
I wonder if this process is worth documenting in the DCR main repo in `docs/architecture` or `docs/commercial`

Reply:
I've created a new file in `docs/commercial`